### DISCUSSION TITLE
Generate random, strong token

### DISCUSSION THREAD

**Discussion by lu-zen** [directus/directus#4820] [Feature Requests]
Hey;

I think a good idea is to auto generate a strong token by default instead of leaving it up to the user to choose one, so we can click and a strong token is generated on backend.

**Comment by benhaynes** [comment]
Agree. Not sure if/how we could make the standard `text input` interface support this (or other custom actions), or if we should just create a new interface dedicated to this.

**Reply by tylerforesthauser** [reply]
Maybe some kind of new interface that allows for general generative content? I'm thinking things like tokens or slugs based on other fields.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
